src/d3a/d3a_core/exceptions.py

```diff
@@ -63,3 +63,7 @@ class InvalidBalancingTradeException(MarketException):
 
 class DeviceNotInRegistryError(MarketException):
     pass
+
+
+class D3ARedisException(D3AException):
```

Comment:
Cool! Can you please also use this exception type for the exceptions added in 1479? Thanks!

Reply:
Will do, thanks!